SCClassLibrary/Common/Collections/linux/extString_linux.sc

```diff
@@ -1,6 +1,26 @@
 + String {
-	runInTerminal {|shell="sh"|
-		("xterm -hold -T 'SuperCollider runInTerminal' -e" + shell + "-c" + this.shellQuote).unixCmd;
+
+	runInTerminal { |shell="/bin/sh"|
+		LinuxPlatform.runInTerminalCmd ?? { LinuxPlatform.runInTerminalCmd = LinuxPlatform.findTerminalEmulator };
+		if(LinuxPlatform.runInTerminalCmd.isNil) {
+			"String:runInTerminal couldn't find a supported terminal emulator. Please set LinuxPlatform.runInTerminalCmd, see Help for details".error;
+			^ this;
+		} {
+			var fpath = PathName.tmp +/+ "SCStringForTerminal%.sh".format(this.hash.asString);
```

Comment:
why "string for terminal" when the function is called "runInTerminal"? obvious names are better

Reply:
`SCStringForTerminal` is used in macOS implementation. I agree that `runInTerminal` would be better. (macOS can be changed later)

Reply:
ah (re this and the comment above) -- i forgot there was a macOS impl. in that case, anything that's here to conform to the macOS impl can stay the way it is, i don't mind. sorry for the confusion!